pkgs/development/libraries/libextractor/default.nix

```
@@ -28,7 +41,12 @@ stdenv.mkDerivation rec {
       sha256 = "01xhcjbzv6p53wz7y2ii76kb8m9iwvnm4ip9w4a0bpgaxqz4b9fw";
       excludes = [ "ChangeLog" ];
     })
-  ];
+  ] ++ stdenv.lib.optional gstreamerSupport (substituteAll {
+    src = ./gst-hardcode-plugins.patch;
+    load_gst_plugins = stdenv.lib.concatMapStrings
+      (plugin: ''gst_registry_scan_path(gst_registry_get(), "${plugin}/lib/gstreamer-1.0");'')
+      (gstPlugins gst_all_1);
+  });
```

Just one missing comment:

```suggestion
  ] ++ stdenv.lib.optionals gstreamerSupport [
    # Libraries cannot be wrapped so we need to hardcode the plug-in paths.
    (substituteAll {
      src = ./gst-hardcode-plugins.patch;
      load_gst_plugins = stdenv.lib.concatMapStrings
        (plugin: ''gst_registry_scan_path(gst_registry_get(), "${plugin}/lib/gstreamer-1.0");'')
        (gstPlugins gst_all_1);
    })
  ];
```

---

Thanks! Committed and force-pushed (to keep only one commit).